pylinalg/__init__.py

```diff
@@ -12,5 +12,10 @@
 version_info = tuple(map(int, __version__.split(".")))
 
 
-from .func import *
-from .obj import *
+from . import matrix, misc, quaternion, vector
+from .matrix import *
+from .misc import *
+from .quaternion import *
+from .vector import *
+
+__all__ = matrix.__all__ + misc.__all__ + quaternion.__all__ + vector.__all__
```

Comment:
I realize now this isn't as clean as it could be

Reply:
> I realize now this isn't as clean as it could be.

How do you mean? 

Is that definition of `__all__` here necessary, since we already make sure we clean `__all__`'s in the submodules? Or is it to help tooling?

Reply:
I fixed it and added a test. If you approve of the changes, we can merge as far as I am concerned.